main.go

```diff
@@ -270,7 +273,7 @@ func (s *service) createProjectCard(issue *github.Issue) {
 	log.Printf("Created a project card. issue=%v card=%v", issue.GetID(), card.GetID())
 }
 
-func (s *service) openNewIssues(issueMap map[string]*github.Issue, comments []*tdglib.ToDoComment) {
+func (s *service) openNewIssues(issueMap map[string]*github.Issue, comments []*tdglib.ToDoComment, newIssueMap *map[string]*github.Issue) {
```

Comment:
Let's move `newIssueMap` to be a field of `service`. Then there will be less juggling around of these pointers.

P.S. There's no need to pass a pointer of map somewhere in Go, because maps are already reference types (like "automatic pointers").